perfkitbenchmarker/disk.py

```diff
@@ -172,6 +172,8 @@ class BaseDiskSpec(spec.BaseSpec):
     mount_point: None or string. Directory of mount point.
     num_striped_disks: int. The number of disks to stripe together. If this is
         1, it means no striping will occur. This must be >= 1.
+    os_type: None or string. OS type of the VM to which the disk will be
```

Comment:
Giving a data disk an os_type feels a bit wrong to me - could we give it a reference to the VM it's attached to (when Attach() is called) and use that VM's os_type instead?